spec/models/like_spec.rb

```diff
@@ -13,6 +13,23 @@
   it 'has a valid factory' do
     expect(FactoryGirl.build(:like)).to be_valid
   end
+  
+  describe '#participation' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.